docs/code/IDataViewTypeSystem.md

```diff
@@ -383,82 +383,53 @@ Key types are instances of the sealed class `KeyType`, which derives from
 
 In addition to its underlying type, a key type specifies:
 
-* A count value, between `0` and `int.MaxValue`, inclusive
+* A count value, between `1` and `ulong.MaxValue`, inclusive
 
-* A "minimum" value, between `0` and `ulong.MaxValue`, inclusive
-
-* A Boolean value indicating whether the values of the key type are contiguous
-
-Regardless of the minimum and count values, the representational value zero
+Regardless of the count value, the representational value zero
```

Comment:
Can we use `ulong.MaxValue` to denote NaN? It can further simplify the conversion between integers and keys. Note that conversion happens in type casting and reading C# native class as `IDataView`.
```suggestion
Regardless of the count value, the representational value zero
```
@TomFinley, what do you think?

Reply:
My original idea comes from having same value for key value and its underlying representation. This way we can simplify the loading/writing value from/to C# data structure. But I got your point, NA'd better to be unique.